homework-submission/d-jason-week2hw/src/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+// import logo from './logo.svg';
+import './App.css';
+import ToDoItems from './ToDoItems';
+import Header from './Header';
+
+class App extends Component {
+  state = {
+    todoList
+  };
+
+  clickHandler = e => {
+    let tickBox = this.state.todoList.slice();
+
+    console.log(e.target.id);
```

Comment:
No `console.log()` in your productions and project unless for testing and then remove it.